src/main/java/org/qiyana/model/Customer.java

```diff
@@ -0,0 +1,57 @@
+package org.qiyana.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Customer {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private String name;
+  private String email;
+  private String address;
+  private String phone;
+
+  @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL, orphanRemoval = true)
+  private List<Order> orders;
+
+  public Customer(Long id, String name, String email, String address, String phone) {
+    this.id = id;
```

Comment:
`id` and any other `GeneratedValue` should never be a constructor parameter. You will pretty much always pass `null` here.